.classpath

```diff
@@ -20,9 +20,10 @@
 	<classpathentry kind="lib" path="lib/common/cmutimelex.jar"/>
 	<classpathentry kind="lib" path="lib/common/en_us.jar"/>
 	<classpathentry kind="lib" path="lib/common/freetts.jar"/>
-	<classpathentry kind="lib" path="lib/common/RXTXcomm.jar">
+	<classpathentry exported="true" kind="lib" path="lib/common/RXTXcomm.jar">
 		<attributes>
 			<attribute name="javadoc_location" value="http://users.frii.com/jarvi/rxtx/doc/"/>
+			<attribute name="org.eclipse.jdt.launching.CLASSPATH_ATTR_LIBRARY_PATH_ENTRY" value="romraider/lib/windows"/>
```

Comment:
This is not required as the JVM command line parameter -Djava.library.path=lib/windows points to the proper location.

Reply:
Removed, this was unintentional actually